test/jit/test_backends.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+import unittest
+
+import torch
+import torch._C
+from torch.testing import FileCheck
+from torch._six import PY2
+
+# Make the helper files in test/ importable
+pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(pytorch_test_dir)
+from torch.testing._internal.jit_utils import JitTestCase
+
+if __name__ == '__main__':
+    raise RuntimeError("This test file is not meant to be run directly, use:\n\n"
+                       "\tpython test/test_jit.py TESTNAME\n\n"
+                       "instead.")
+
+
+def to_test_backend(module, method_compile_spec):
+    return torch._C._jit_to_test_backend(module, {"forward": method_compile_spec})
+
+
+class MyModule(torch.nn.Module):
+    def __init__(self):
+        super(MyModule, self).__init__()
+
+    def forward(self, x, h):
+        new_h = torch.tanh(x + h)
+        return new_h
+
+
+class TestBackends(JitTestCase):
+    def test_simple(self):
+        # Test compile.
+        scripted_module = torch.jit.script(MyModule())
+        lowered_module = to_test_backend(scripted_module._c, {"key": "value"})
+
+        # Test execute.
+        input = torch.randn(5)
+        output = lowered_module.forward(input)
```

Comment:
This call should fail, `forward` should take two arguments. We should have the test backend trivially implement `forward` (calling `torch::tanh` on inputs and returning it), then test that the Python, TS, and lowered module all produce the same result given an input.